target/linux/armvirt/64/target.mk

```diff
@@ -1,8 +1,6 @@
 ARCH:=aarch64
 SUBTARGET:=64
-BOARDNAME:=QEMU ARMv8 Virtual Machine (cortex-a53)
-CPU_TYPE:=cortex-a53
```

Comment:
This will switch the subtarget to generic aarch64 Is this intended?
Have you done some performance tests?

Reply:
I'm not sure having A53 as the default CPU type is the best idea (even though our hardware has them).

There are roughly three Arm64 generations of interest: Armv8.0 (A53,A72, 2014-2017ish), Armv8.2+ (+SVE instructions, most recent Arm64 CPUs), Armv9 (+SV2 instructions, coming ~2024). 
After A55 there are no more in-order designs as well. (See https://en.wikipedia.org/wiki/Comparison_of_Armv8-A_processors)
I could try running benchmarks between the old and new AWS instances to see if choosing a specific CPU types produces an improvement.